incubator/azuremonitor-containers/README.md

```diff
@@ -55,7 +58,8 @@ $ helm install --name myrelease-1 \
 
 ## Uninstalling the Chart
 
-To uninstall/delete the `myrelease-1` release:
+To uninstall/delete the installed azure monitor for container release:
```

Comment:
Also this isn't necessary

Reply:
> Also this isn't necessary

Update the note to cover deprecated flags in helm3